[09:06:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot/dist/vkbot.js:72:35)
[09:06:20] Error: {
  error_code: 10,
  error_msg: 'Internal server error: could not check access_token now, check later.',
  request_params: [Array]
}
[15:30:27] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot/dist/vkbot.js:72:35)
[15:30:27] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
